tests/apply-shim.html

```diff
@@ -138,6 +138,22 @@
     </style>
     </custom-style>
   </template>
+  <template id="important">
+    <style>
+    :host {
+      --mixin-important: {
+        border: 2px solid black !important;
```

Comment:
Could you add another property which does not have important in both mixins and assert it takes the value of the last one?